kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/facility-permissions-form/index.vue

```diff
@@ -141,8 +141,8 @@
     $trs: {
       facilityPermissionsSetupFormHeader: 'Choose a Facility setup',
       facilityPermissionsSetupFormDescription:
-        'How will you be using Kolibri? You can customize these settings later.',
-      facilityPermissionsPresetDetailsLink: 'Setup details',
+        'How will you be using Kolibri? (You can customize these settings later)',
+      facilityPermissionsPresetDetailsLink: 'More information',
       facilityPermissionsPresetDetailsHeader: 'Facility setup details',
```

Comment:
The only string change in this PR where I would disagree, every other instance has been a great improvement... :slightly_smiling_face: 

For accessibility reasons, `Facility setup details` gives more context to the link then just `More information`. If we want to keep the latter, we need to add another string/span **inside the link itself**, with `visuallyhidden` class applied, to complement the context for screenreader users. Final link should look something like this:

```<a>More information <span class="visuallyhidden">about Facility setup options</span>.</a>```